js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,90 @@
+//  Patrice Briggs
+//  Homework Module 6
+
+//test suite for enhancedDate function
+describe("enhancedDate", function () {    
+    'use strict';
+
+    //test setDate function
+    describe("setDate", function () {
+        it("should return a new Date Object if nothing passed in", function () {
+        	expect(enhancedDate.setDate()).to.be(new Date());
```

Comment:
`be()` isn't a valid assertion function - you could try `a()` or `an()`, but in this case I'd recommend using instanceof instead.